libraries/stdlib/genosl/mx_transformpointM.inline

```diff
@@ -1 +1 @@
-transform({{mat}}, (point){{in}})
+transform({{mat}}, {{in}})
```

Comment:
I "think" that this will use the vector type since that is the code produced. Want to double check.

Reply:
Yep, looks good. Since "in" always has the MaterialX type it could never be OSL's type "point" or "normal" here.

Reply:
As per my comment above I think the source code file could be renamed to "mx_transformmatrix.inline" for consistency.